src/bindings/c/include/openvino/c/ov_model.h

```diff
@@ -32,44 +32,128 @@ typedef struct ov_model ov_model_t;
 OPENVINO_C_API(void) ov_model_free(ov_model_t* model);
 
 /**
- * @brief Get the outputs of ov_model_t.
+ * @brief Get a const input port of ov_model_t.
  * @ingroup model
  * @param model A pointer to the ov_model_t.
- * @param output_nodes A pointer to the ov_output_nodes.
+ * @param input_port A pointer to the ov_output_const_port_t.
  * @return Status code of the operation: OK(0) for success.
  */
-OPENVINO_C_API(ov_status_e) ov_model_outputs(const ov_model_t* model, ov_output_node_list_t* output_nodes);
+OPENVINO_C_API(ov_status_e) ov_model_const_input(const ov_model_t* model, ov_output_const_port_t** input_port);
 
 /**
- * @brief Get the outputs of ov_model_t.
+ * @brief Get a const input port of ov_model_t by name.
  * @ingroup model
  * @param model A pointer to the ov_model_t.
- * @param input_nodes A pointer to the ov_input_nodes.
+ * @param tensor_name input tensor name (char *).
+ * @param input_port A pointer to the ov_output_const_port_t.
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e)
+ov_model_const_input_by_name(const ov_model_t* model, const char* tensor_name, ov_output_const_port_t** input_port);
+
+/**
+ * @brief Get a const input port of ov_model_t by port index.
+ * @ingroup model
+ * @param model A pointer to the ov_model_t.
+ * @param index input tensor index.
+ * @param input_port A pointer to the ov_output_const_port_t.
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e)
+ov_model_const_input_by_index(const ov_model_t* model, const size_t index, ov_output_const_port_t** input_port);
```

Comment:
I think we also need to have:
`ov_model_inputs_size()` and `ov_model_outputs_size()` to provide information about number of inputs/outputs in the model

Reply:
@RICKIE777 please add new interfaces as Ilya suggestion,

OPENVINO_C_API(ov_status_e) ov_model_inputs_size(const ov_model_t* model);
OPENVINO_C_API(ov_status_e) ov_model_outputs_size(const ov_model_t* model);

Then we can set right index to get the port, such as:
        ov_model_const_input_by_index(model, index, &input_port);

Reply:
Update it in new commit. Please review.